src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -41,7 +41,7 @@ public GitHubAccessor(Config config)
 
             if (!string.IsNullOrEmpty(config.Secrets.GithubToken))
             {
-                _httpClient = new HttpClient(_httpClientHandler);
+                _httpClient = new HttpClient(_httpClientHandler); // lgtm [cs/httpclient-checkcertrevlist-disabled]
```

Comment:
Would `_httpClient = new HttpClient(new HttpClientHandler() { CheckCertificateRevocationList = true })` suppress the issue?

Reply:
I suppose so, but we would have another warning [CA2000](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2000). We met this when we checked in this code.

Reply:
We could add `dotnet_code_quality.CA2000.excluded_symbol_names = HttpClientHandler` to `.editorconfig` per

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2000